src/commands/utils/config.ts

```diff
@@ -38,21 +38,39 @@ export function getKustomizeConfig(): Errorable<KustomizeConfig> {
 export function getKubeloginConfig(): Errorable<KubeloginConfig> {
     const kubeloginConfig = vscode.workspace.getConfiguration('azure.kubelogin');
     const props = combine([getConfigValue(kubeloginConfig, 'releaseTag')]);
- 
+
     if (failed(props)) {
-       return {
-          succeeded: false,
-          error: `Failed to read azure.kubelogin configuration: ${props.error}`
-       };
+        return {
+            succeeded: false,
+            error: `Failed to read azure.kubelogin configuration: ${props.error}`
+        };
     }
- 
+
     const config = {
-       releaseTag: props.result[0]
+        releaseTag: props.result[0]
     };
- 
-    return {succeeded: true, result: config};
- }
- 
+
+    return { succeeded: true, result: config };
+}
+
+export function getKubectlGadgetConfig(): Errorable<KubeloginConfig> {
+    const kubeloginConfig = vscode.workspace.getConfiguration('azure.kubectlgadget');
+    const props = combine([getConfigValue(kubeloginConfig, 'releaseTag')]);
```

Comment:
Just noticed something I failed to spot in the `getKubeloginConfig` function, that also applies here. It doesn't make sense to me that we're `combine`-ing the results of an array containing a single item...

Would it work if we changed both this and the kubelogin function to:
```javascript
const config = getConfigValue(kubeloginConfig, 'releaseTag');
if (failed(config)) {
return {
    succeeded: false,
    error: `Failed to read azure.kubectlgadget configuration: ${props.error}`
};

return { succeeded: true, result: config.result };
```